webserver.py

```diff
@@ -266,9 +309,11 @@ def enqueue(self, youtube_id):
 
 	@cherrypy.expose
 	@cherrypy.tools.json_out()
-	def list(self):
+	def status(self):
 		elems = self.elems()
-		return {"listing": elems, "titles": self.titles(set(elem["ytid"] for elem in elems)), "volume": get_volume()}
+		raw_status = redis.get("musicastatus")
+		playback_status = json.loads(raw_status.decode()) if raw_status else {}
+		return dict(playback_status, **{"listing": elems, "titles": self.titles(set(elem["ytid"] for elem in elems)), "volume": get_volume()})
```

Comment:
Couldn't you just make changes to playback_status, since it's a new copy of a dict? This seems like a strange way to handle adding entries to a dictionary.

Reply:
Sure, that'd also work fine.